.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+* @TokoBapak/frontend
+*.nomad @TokoBapak/infrastructure
```

Comment:
The infrastructure team doesn't have access to this repo I think

Reply:
nanti tinggal ditambahin